test/src/test/java/jenkins/model/NodesTest.java

```diff
@@ -237,8 +237,6 @@ public long check(SlaveComputer c) {
     }
 
     private static class InvalidNode extends Slave {
-        // JEP-200 whitelist changes prevent this field (and thus instances of this class) from being serialized.
-        private ClassLoader cl = InvalidNode.class.getClassLoader();
```

Comment:
This is needed to pass the tests in this class. It's not unused, it breaks the serialization, hence the class name and comment.